src/aux-server/aux-web/aux-player/PlayerHome/PlayerHome.ts

```diff
@@ -182,6 +183,8 @@ export default class PlayerHome extends Vue {
 
     private _simulations: Map<BrowserSimulation, Subscription>;
 
+    private generatedName: string = '';
```

Comment:
@TroyceGowdy By convention, all variables that are accessed in the Vue code should have `public` visibility (which is the default) instead of `private` visibility. (See [Member Visibility](https://www.typescriptlang.org/docs/handbook/2/classes.html#member-visibility) from the TypeScript documentation)

Variables that aren't accessed by the Vue code can be `private`, and when we decide to mark them `private` then we also always start the variable name with an underscore.

In this case, because `generatedName` is used by Vue code, it should be public and not have an underscore.